4-toolchain/Dockerfile

```diff
@@ -47,12 +46,12 @@ WORKDIR ${HOME}
 
 # Use entrypoint so path is correctly adjusted already at the time the command
 # is searching, so something like docker run IMG gcc runs binary from SCL.
-COPY contrib/bin/container-entrypoint /usr/bin/container-entrypoint
+COPY root/opt/app-root/bin/container-entrypoint /usr/bin/container-entrypoint
```

Comment:
We could just make it a symlink inside the `root` subdirectory in this repo so that it gets copied to the image without needing a separate Docker instruction.